src/service/world-api-adapter.js

```diff
@@ -223,6 +219,24 @@ module.exports = function (config) {
         },
 
         /**
+        * Set the filter for the current instance of the world adapter
+        *
+        * **Example**
+        * var ws = new F.service.World({...}).load('123').addUser({ userId: '123' });
+        *
+        *
+        */
+        load: function (worldId) {
+            if (!worldId || typeof worldId !== 'string') {
+                throw new Error('load needs a worldId string to load (' + worldId + ')');
```

Comment:
While it's somewhat useful for nulls I would probably be more confused if I saw 'load needs a worldId string to load (0)' or 'load needs a worldId string to load (false)' and had to guess what the stuff in brackets meant :). Maybe change it to "..string to load, received: (typeof worldId)", or just remove the worldId part of it? Minor point either way.